pydantic/networks.py

```diff
@@ -722,7 +722,7 @@ def _build_pretty_email_regex() -> re.Pattern[str]:
     name_chars = r'[\w!#$%&\'*+\-/=?^_`{|}~]'
     unquoted_name_group = rf'((?:{name_chars}+\s+)*{name_chars}+)'
     quoted_name_group = r'"((?:[^"]|\")+)"'
-    email_group = r'<\s*(.+)\s*>'
+    email_group = r'<\s*(.+)>'
```

Comment:
```suggestion
    email_group = r'<(.+)>'
```

The `.strip()` call also removes leading whitespaces.